spec/datadog/appsec/processor_spec.rb

```diff
@@ -246,7 +246,8 @@
 
     let(:input) { input_scanner }
 
-    subject(:context) { described_class.new.new_context }
+    subject(:processor) { described_class.new }
+    subject(:context) { processor.new_context }
```

Comment:
I am a bit confused here, could you elaborate about the change, because the change seems to be same to me.

As far as I know, `subject` is a test helper to generate a test subject for a test example, it is kinda weird to have multiple `subject` statements setup in a single context.

Reply:
Fixed

Reply:
Reason is here: https://github.com/DataDog/libddwaf/issues/76#issuecomment-1134750143